src/applications/disability-benefits/all-claims/pages/addConditionsPages.js

```diff
@@ -0,0 +1,373 @@
+import React from 'react';
+import { arrayBuilderPages } from 'platform/forms-system/src/js/patterns/array-builder';
+import {
+  arrayBuilderItemFirstPageTitleUI,
+  arrayBuilderItemSubsequentPageTitleUI,
+  arrayBuilderYesNoSchema,
+  arrayBuilderYesNoUI,
+  currentOrPastMonthYearDateSchema,
+  currentOrPastMonthYearDateUI,
+  radioSchema,
+  radioUI,
+  selectSchema,
+  selectUI,
+  textareaUI,
+  textUI,
+  titleUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+
+import Autocomplete from '../components/Autocomplete';
+import {
+  ConditionInstructions,
+  DuplicateAlert,
+  ServiceConnectedDisabilityDescription,
+} from '../content/addConditions';
+import disabilityLabelsRevised from '../content/disabilityLabelsRevised';
+import { missingConditionMessage } from '../validations';
+
+const causeOptions = {
+  NEW:
+    'My condition was caused by an injury or exposure during my military service.',
+  SECONDARY:
+    'My condition was caused by another service-connected disability I already have. (For example, I have a limp that caused lower-back problems.)',
+  WORSENED:
+    'My condition existed before I served in the military, but it got worse because of my military service.',
+  VA:
+    'My condition was caused by an injury or event that happened when I was receiving VA care.',
+};
+
+const createCauseFollowUpTitles = formData => {
+  const { condition, cause } = formData;
+  const conditionTitle = condition || 'condition';
+
+  const causeTitle = {
+    NEW: `Details of injury or exposure that caused ${conditionTitle}`,
+    SECONDARY: `Details of the other condition that caused ${conditionTitle}`,
+    WORSENED: `Details of the injury or exposure that worsened ${conditionTitle}`,
+    VA: 'Details of the injury or event in VA care',
```

Comment:
For consistency I think we should find a way to add the condition name to the VA care title. When I was going through I was expecting the condition name in the title. I know we are editing one condition at a time, but you have to backtrack to doublecheck what condition you're on.